lib/_.coffee

```diff
@@ -1,50 +0,0 @@
-_ = {}
```

Comment:
since we mosly work with y-lists, we actually don't need this sugar for Arrays. There was only reference, where you used `_.any` (I replaced it accordingly). We can re-include this, when we actually need it.